shared/middleware/helpers/bucket_helper.rb

```diff
@@ -24,6 +24,8 @@ def category_from_file_type(extension)
     mime_type = Sinatra::Base.mime_type(extension)
     if mime_type == 'application/pdf'
       'pdf'
+    elsif ['.doc', '.docx'].include? extension
+      'doc'
```

Comment:
I suppose this is a lot better than `application/vnd.openxmlformats-officedocument.wordprocessingml.document`.